pkg/tfbridge/provider_test.go

```diff
@@ -4536,7 +4536,138 @@ func TestPlanResourceChangeStateUpgrade(t *testing.T) {
 	}`)
 }
 
+func TestUnknowns(t *testing.T) {
+	// Related to [pulumi/pulumi-terraform-bridge#1885]
+	p := &schemav2.Provider{
+		Schema: map[string]*schemav2.Schema{},
+		ResourcesMap: map[string]*schemav2.Resource{
+			"example_resource": {
```

Comment:
Doesn't seem to cover blocks (Elem: &schemav2.Resource{..}), but only attributes. Blocks are indeed the interesting cases. Would it be possible to cover?

Reply:
mb, thanks for noticing.